2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+Title: Funding Freedom Society - Idee zur Finanzierung freier Softwareprojekte
+Date: 2018-03-04 9:00
+Category: Artikel
+Tags: Freie Software, Gesellschaft
+Slug: FFS-wp
+Author: Carsten
+Summary: Finanzierungsmodell zur Steigerung von Qualität und Verbreitung von FLOSS durch
+Kombination existierender Geschäftsmodelle (Spenden, Merchandise, Bounties, Funding-Organisationen.)
+
+---
+
+# Funding Freedom Society
+
+Dieser Text beschreibt eine Idee zur Steigerung von Qualität und Verbreitung von freier quelloffener Software ([FLOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software)) und freien dezentralen Web-Diensten.
+Er geht nicht darauf ein, *warum* diese Ziele erstrebenswert sind.
+Er ist eine stark inspiriert durch [Colin Percivals Plan für Open Source Software Maintainer](http://www.daemonology.net/blog/2017-05-11-plan-for-foss-maintainers.html).
+
+## Bestandsaufnahme
+
+
+Ausgangslage: Mit zunehmender Digitalisierung wird Software privat und beruflich immer wichtiger.
+Freie quelloffene Software für Endbenutzer*innen ist XX Jahre nach Etablierung des Begriffes OpenSource immer noch (oder mehr denn je) ein Nischenprodukt.
+Gründe: Mangelnde ...
+
+- Bekanntheit der Produkte und ihrer Vorteile
+- Usability
+- Features
+- Stabilität
+- Dokumentation
+
+ Mit freien dezentralen Diensten (z.B. Jabber als Ersatz für WhatsApp) verhält es sich ähnlich.
+ Der Einfachheit halber geht der folgende Text erstmal nur auf "klassische Software" ein,
+ inbesondere Desktop-Applikationen.
+
+
+**Warum ist proprietäre Software in den genannten Punkten oft besser?**
+Geschäftsmodell (Verkauf von Software uder Daten) generiert Einnahmen.
+Dafür kann Arbeitskraft und Aufmerksamkeit eingekauft werden.
+
+**Warum funktioniert das bei Freier Software so nicht?**
+Diese kann auf Grund der wesensbestimmenden Freiheit nicht "verkauft" werden.
```

Comment:
"It depends". Conversations verkauft sich über den Playstore, ich weiß aber nicht wieviel Prozent der Einnahmen Playstore-Verkäufe vs. Account-Abos sind.

Reply:
@iNPUTmice wird sicher gerne als Ansprechpartner herhalten um das am Bsp. näher zu beleuchten.

Reply:
fwiw es findet kein cross financing zwischen den Abos des Servers (conversations.im) und der App Conversations statt.
Die Abos für den Server zahlen für den Serverbetrieb und die Weiterentwicklung im Server Bereich und die PlayStore Verkäufe zahlen für die App.

Reply:
Wie Jonas schon mal gesagt hat: Conversations ist eines der wenigen Beispiele, wo es geklappt hat. Aber es ist fraglich, ob es mit einer Desktop-Applikation auch gehen würde.
Der App-Markt auf mobilen Geräten ist anders gestrickt als auf dem Desktop.

Also Zustimmung zu "it depends", aber weiter oben steht ja, dass der Text sich auf Endanwender*innen und Desktop-Software bezieht.

Reply:
Ob das auf dem Desktop nicht klappen würde weiß ich nicht. Das müsste mal jemand ausprobieren. Mittlerweile haben auch Windows und Mac OS integrierte stores. Gerade unter Windows wo das Kompilieren von Software notorisch schwierig ist könnte das mit dem Verkaufen durchaus klappen.

Reply:
Hmm. Also vielleicht habe ich da zu sehr die Linux-Brille auf, jedenfalls habe ich hauptsächlich diese Plattform im Kopf. Und da würde ich es massiv befremdlich finden, wenn das Paketsystem durch einen Appstore ersetzt würde. Freiwillig und selbstbestimmt würde ich dagegen gerne was bezahlen.

Reply:
Ich bin mir da garnicht so sicher. Ich stelle mir da ein Pay What You Want mit einem vorgeschlagenen Preis (ggfs. halt 0 EUR) vor. Wenn das in die Paketverwaltung integriert wäre, wäre die Einstiegshürde auch geringer. Ein Ottonormalnutzer müsste ja erstmal herausfinden welchen Spendenservice (falls überhaupt) die spezielle Software nun nutzt, da ggfs. einen Account erzeugen etc. etc..

Freiwillige Abo-Modelle ala Patreon die ggfs. automatisch mit deinstallation beendet werden, plus eine zentrale Übersicht darüber welchen Paketen man wieviel spendet. Wäre doch nice.

Das deckt natürlich Pakete die es im Store nicht gibt nicht ab. Das dürfte aber die wenigsten normalen Endbenutzer betreffen.

(Und wenn das System so implementiert ist, dass es einfach mit mehrenen Spendenservices als Backends reden kann und die Informationen aggregiert, dann kann man auch problemlos externe "Pakete" hinzufügen. Sweet!)

Reply:
0 ist für Einflussnahme irgendwie komisch; ich bin für Subscription (Informationen) zum Nulltarif. Aber Voting exklusiv für Unterstützer. Auch sollte man die Kosten betrachten, z.B. Überweisungseingänge verursachen gerne Kosten, auch solche mit Kleinbeträgen; daher wird sich ein Mindestbetrag ergeben.
Zum Auftakt könnte man leichter werben mit "die erste Million Nutzer kostenfrei" o.ä. – aber dazu dann später mal mit einem Marketingspezialisten reden.

Reply:
> 0 ist für Einflussnahme irgendwie komisch; ich bin für Subscription (Informationen) zum Nulltarif. Aber Voting exklusiv für Unterstützer.

Klingt vernünftig.